static/src/javascripts/bootstraps/enhanced/common.js

```diff
@@ -139,7 +139,8 @@ define([
 
             initialiseStickyHeader: function () {
                 if (config.switches.viewability
-                    && !(config.page.isProd && config.page.contentType === 'Interactive')
```

Comment:
Not sure why we only disabled the sticky nav for interactive pages in prod. @uplne, any ideas? (Re. https://github.com/guardian/frontend/commit/8b5f2ddd15cfa2e85cf460d20a5286b4b01558a4)